src/ServiceControlInstaller.Engine/Instances/ServiceControlNewInstance.cs

```diff
@@ -151,7 +151,7 @@ public void RegisterService()
                 ServiceDescription = ServiceDescription
             };
             var dependencies = new List<string>();
-            if (TransportPackage.Equals("MSMQ", StringComparison.OrdinalIgnoreCase))
+            if (TransportPackage.ZipName.Equals("MSMQ", StringComparison.OrdinalIgnoreCase))
```

Comment:
Why compare the ZipName and not the transport name here?

Reply:
Transport name will change to include the topology (e.g. `RabbitMQ - Forwarding Topology`) so comparing the ZipName lets the validation logic work for all types of rabbit mq transports